.github/workflows/integration_tests.yml

```diff
@@ -1,18 +1,14 @@
 name: Integration tests
-on:
-  pull_request: {}
-  push:
-    paths-ignore:
-    - '*.md'
-    - '**/*.md'
-    - 'web/app/package.json'
```

Comment:
You can still leave these to avoid triggering on doc changes PRs? (same comment for the other workflows)

Reply:
Note that this was only set on `push` and didn't apply to pull_requests. I think we'd either have to stop requiring this workflow via branch protections or we'll have to admin merge all PRs that only touch these files. Otherwise the branch will be missing a required check.

Reply:
I forgot about requiring workflows... this looks good then :+1: